ci/tasks/test-rake-task.sh

```
@@ -226,3 +234,15 @@ pushd bosh/src
 
   bundle exec rake --trace "${RAKE_TASK}"
 popd
+
+if [ -d release-details ]; then
+  if [ -f bosh/src/tmp/parallel_runtime_rspec.log ]; then
+    mkdir -p release-details/parallel_spec_runtimes
+    cp bosh/src/tmp/parallel_runtime_rspec.log "release-details/parallel_spec_runtimes/integration_${DB}_${UPDATE_VM_STRATEGY:-delete-create}.log"
+    cd release-details
+    git config user.name "CF-Bosh CI-Bot"
+    git config user.email "cf-bosh-ci-bot@localhost"
+    git add parallel_runtime_rspec_${DB}_${UPDATE_VM_STRATEGY:-delete-create}.log
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)